schema/resolvers.js

```diff
@@ -3,6 +3,7 @@ const { includes } = require("lodash");
 
 const getId = args => args.newId || args.id;
 const getNewId = entity => entity.id.toString(10);
+const undelete = obj => Object.assign({}, obj, { state: "Active" });
```

Comment:
I wonder whether we should just go ahead and add an undelete method to Repository objects? Might be a bad idea, thinking aloud

Reply:
Personally, I prefer to keep repositories as CRUD as possible (with exception of Find and FindAll - this is similar to how I've seen them in other frameworks. It feels like undeleting belongs at a "Service" level above the repository.